operator/controllers/internal/csiscaleoperator/csiscaleoperator_package.go

```diff
@@ -363,7 +363,7 @@ func (c *CSIScaleOperator) GenerateSnapshotterClusterRole() *rbacv1.ClusterRole
 			{
 				APIGroups: []string{snapshotStorageApiGroup},
 				Resources: []string{volumeSnapshotContentsResource},
-				Verbs:     []string{verbCreate, verbGet, verbList, verbWatch, verbUpdate, verbDelete, verbPatch},
```

Comment:
why are create and delete permissions removed? Is it not needed?

Reply:
yes they are not needed with new snapshotter images